### DISCUSSION TITLE
Change the Name of plugin from AspireUpdate to Aspirepress

### DISCUSSION THREAD

**Discussion by harikrishnanr994** [aspirepress/AspireUpdate#144] [General]
The name "aspireupdate" for the plugin could be a bit misleading to the users or devs who will be installing it.
The aim of the plugin is to be a distribution channel to install and manage the wp assets and not just to update the plugins. Aspireupdate may mislead devs to think it is kind of auto update feature.
Also this is the only plugin/endpoint that most of the devs/users will interact. So the plugin name is a big Marketing/branding thing.

**Comment by namithj** [comment]
#140 There is this happening, but we should all move from that discussion to a broader one.  We have already renamed the plugin once, lets finalize and make sure we don't end up renaming it again after this one as it affects translations at this point.

**Reply by asirota** [reply]
This is a bigger discussion. Branding needs to get involved

**Comment by asirota** [comment]
I've shared this to #branding channel.

**Comment by afragen** [comment]
Just to let everyone know GlotPress changes slugs to all lowercase. Consequently language files will be in the format `aspireupdate-{$locale}.po`.

**Reply by asirota** [reply]
Plugin is not changing names for now.

**Comment by afragen** [comment]
Even the Plugin Check plugin looks to the name of the plugin folder as the slug.

https://github.com/WordPress/plugin-check/blob/608225ab2507ac393d94c3980f132c8283b9dc1a/includes/Plugin_Context.php#L84

Another reason to change the repo to lowercase.

**Comment by asirota** [comment]
this is a dupe of #140
